clients/extension/styles/lighthouse.css

```diff
@@ -43,168 +52,122 @@ html, body {
 }
 
 .header__icon {
-  margin-right: 16px;
+  margin: 2px auto 0;
 }
 
-.header-titles {
-  display: flex;
-  flex-direction: column;
-  align-self: center;
-  flex: 1;
-  overflow: hidden;
-  text-overflow: ellipsis;
+.errormsg {
+  color: var(--color-blue);
+  font-weight: 500;
+  text-align: center;
+}
+.errormsg:empty {
+  display: none;
 }
 
-.header-titles__main {
-  margin: 0;
-  font-family: 'Roboto', Arial, sans-serif;
-  color: currentColor;
-  font-weight: 300;
-  margin-bottom: 4px;
+.button:disabled {
+  background-color: #cccccc;
+  border: none;
+  color: #666666;
+  cursor: not-allowed;
 }
 
-.header-titles__url {
-  margin: 0;
-  font-size: 14px;
-  opacity: 0.74;
-  color: currentColor;
+.button--generate {
+  cursor: pointer;
   font-family: 'Roboto', Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
-  font-weight: 400;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+  font-weight: 500;
+  font-size: var(--font-size);
+  border: none;
+  color: var(--color-bg);
+  margin: 8px auto;
+  padding: 6px 14px 6px 14px;
+  border-radius: 13px;
+  background-color: var(--color-blue);
 }
 
-.header-titles__throttling {
-  position: relative;
-  margin-top: 5px;
-  padding-left: 20px;
+.section {
+  display: grid;
+  justify-content: center;
 }
 
-.header-titles__throttling input {
-  position: absolute;
-  left: 0;
+.section:first-child {
+  border-bottom: solid 1px lightgray;
+  padding-bottom: 10px;
 }
 
-.main {
-  height: 70px;
-  display: flex;
-  padding: 16px;
-  background-color: var(--lh-blue-color);
-  justify-content: space-around;
-  align-items: flex-end;
+.devtools-header {
+  font-size: var(--font-size);
+  font-weight: 500;
+  margin-bottom: 5px;
 }
 
-.button {
-  cursor: pointer;
-  font-family: 'Roboto', Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  font-weight: 500;
-  font-size: 12px;
-  border: none;
-  color: #fff;
-  background-color: #2979ff;
-  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.50);
-  border-radius: 2px;
-  padding: 7px 12px 5px 12px
+.devtools-note {
+  font-size: 14px;
+  color: grey;
 }
 
-.button--generate {
-  font-weight: 500;
-  font-size: 16px;
-  padding: 8px 16px 10px 16px;
-  align-self: stretched;
-  margin-left: auto;
+.devtools-shortcut {
+  font-family: monospace;
```

Comment:
I think yuin today indicated some style details?  

lets carry over this beautiful font stack from LH: 

```css
  --report-font-family-monospace: 'Roboto Mono', 'Menlo', 'dejavu sans mono', 'Consolas', 'Lucida Console', monospace;
```